lighthouse-core/computed/metrics/timing-summary.js

```diff
@@ -45,6 +46,7 @@ class TimingSummary {
     const largestContentfulPaint = await requestOrUndefined(LargestContentfulPaint, metricComputationData); // eslint-disable-line max-len
     const firstCPUIdle = await requestOrUndefined(FirstCPUIdle, metricComputationData);
     const interactive = await requestOrUndefined(Interactive, metricComputationData);
+    const cumulativeLayoutShift = await CumulativeLayoutShift.request(trace, context); // eslint-disable-line max-len
```

Comment:
```suggestion
    const cumulativeLayoutShift = await CumulativeLayoutShift.request(trace, context);
```
looks like it doesn't need it

Reply:
also, I think this will want the `requestOrUndefined` thing the others use since the computed artifact looks like it either returns a value or throws, and we shouldn't ditch `timing-summary` if it does throw, so maybe it's

```suggestion
    const cumulativeLayoutShift = await requestOrUndefined(CumulativeLayoutShift, trace);
```
?